go.mod

```diff
@@ -3,19 +3,45 @@ module github.com/Portshift/dockle
 go 1.13
 
 require (
-	github.com/containers/image/v5 v5.1.0
+	github.com/Portshift/klar v1.9.10-0.20201117231249-97be18dd7f8a
```

Comment:
@akpsgit this should point to master, no?

Reply:
Of course :) Wanted to review them in parallel :)